static/website/templates/get-question.html

```diff
@@ -90,9 +96,13 @@
         {% endif %}
 
         <span class="modify">
-            {% if user|can_edit:question and not ans_count or user|is_moderator and MODERATOR_ACTIVATED %}
+            {% if user|can_edit:question and not ans_count or MODERATOR_ACTIVATED %}
```

Comment:
The logic here is:

```
if user|can_edit:question and not ans_count:
   show the edit link
else if MODERATOR_ACTIVATED:
   show the edit link
```

so ans_count is checked only for the first part. is this logic correct?

Reply:
Yes, this is correct.
not ans_count is being checked only for second part.
Moderator can edit the question even if some answers are posted.
But user can edit only when no answer is being posted

Reply:
okay, thanks for all the confirmations.